### DISCUSSION TITLE
Suggestion: Default to Disable 'Enable new sign ups'

### DISCUSSION THREAD

**Discussion by harrylisen** [open-webui/open-webui#6662] [Ideas]
Many people are unaware that this is enabled by default, and some take advantage of this to register and use other people's webui.

**Comment by mikahe** [comment]
A side note from our lab testing (0.6.36) env related to 'Enable New Sign Ups' (feature or bug). 

For now we want to have 'Enable New Sign Ups' enabled. We also configured LDAP (and later on will also use OAUTH).

Now the interesting finding is that if first login creates new email account (not using LDAP), 'Enable New Sign Ups' goes disabled.
When first login is done with LDAP, 'Enable New Sign Ups' stays enabled.

**Comment by athoik** [comment]
Hi, 

Check the [ENABLE_LOGIN_FORM](https://docs.openwebui.com/getting-started/env-configuration/#enable_login_form).
